version.php

```diff
@@ -24,8 +24,8 @@
 
 defined('MOODLE_INTERNAL') || die;
 
-$plugin->version   = 2021051000;
+$plugin->version   = 2021081900;
```

Comment:
should we add the same warning here?

Reply:
I don't think so. 310 branch seems to be the main branch for Moodle 3.10 and onwards, so standard versioning is acpected here (when version number is bumped up to the current date)

Reply:
well... it's the main branch now, but won't be in November when 4.0 is out. How will we remember to do it for this branch again? Should we document it somewhere else?

Reply:
We will discuss the best approach the next internal dev  meeting as it should be applied to all plugins. So good to go as is for now.